examples/cart/index.js

```diff
@@ -158,13 +160,40 @@ $(function() {
   ============================================================ */
   function openCart() {
     $('.cart').addClass('js-active');
+    $('.overlay').addClass('js-active');
   }
 
   /* Close Cart
   ============================================================ */
   function closeCart() {
+    $('.cart').removeClass('js-active');
+    $('.overlay').removeClass('js-active');
+  }
+
+  /* Close Cart Handler
+  ============================================================ */
+  function closeCartHandler() {
     $('.cart .btn--close').click(function () {
-      $('.cart').removeClass('js-active');
+      closeCart();
+    });
+  }
+
+  /* Overlay Handler
+  ============================================================ */
+  function overlayHandler() {
+    $('.overlay').click(function () {
+      closeCart();
+    });
+  }
+
+  /* Key Press Handler
+  ============================================================ */
+  function keyPressHandler() {
+    $(document).keydown(function(e){
+       var code = e.keyCode || e.which;
+       if (code === 27) {
```

Comment:
Make this a constant somewhere. Like: `var ESCAPE_KEYCODE === 27`, then `if (code === ESCAPE_KEYCODE)`. Try to stay away from magic numbers.